pages/en/supported-networks/near.mdx

```diff
@@ -62,6 +62,13 @@ dataSources:
     source:
       account: app.good-morning.near # This data source will monitor this account
       startBlock: 10662188 # Required for NEAR
+      accounts: 
+        prefixes:
+          - app
+          - good
+        suffixes:
+          - morning.near
+          - morning.testnet
```

Comment:
I think remove the `testnet` from here, as it's mixing networks which isn't technically possible, so this is potentially confusing